mne/beamformer/_lcmv.py

```diff
@@ -39,33 +50,46 @@ def _reg_pinv(x, reg):
     return linalg.pinv(x), d
 
 
-def _setup_picks(picks, info, forward, noise_cov=None):
-    if picks is None:
-        picks = pick_types(info, meg=True, eeg=True, ref_meg=False,
-                           exclude='bads')
+def _setup_picks(info, forward, data_cov=None, noise_cov=None):
```

Comment:
"""Add a mini 1 line docstring here that is a grammatically complete sentence.""""